source/sass/global/fonts.scss

```
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: "oswaldregular";
+  font-weight: 400;
+  font-style: normal;
+  src: local("oswaldregular"),
+  url("../../fonts/oswaldregular.woff2") format("woff2"),
+  url("../../fonts/oswaldregular.woff") format("woff");
```

Путь мы смотрим не относительно этого файла а относительно выходного файла css. А там чтобы его найти надо подняться только на 1 уровень вверх 
```suggestion
  url("../fonts/oswaldregular.woff") format("woff");
```